site/_docs/installation.md

```diff
@@ -103,4 +103,25 @@ Check out [the extras page](../extras/) for more information.
   </p>
 </div>
 
-Now that you’ve got everything installed, let’s get to work!
+## Already Have Jekyll?
+
+Before you start developing with Jekyll, you may want to check that you're up to date with the latest version. To find your version of Jekyll, run one of these commands:
+
+```sh
+$ jekyll --version
+$ gem list jekyll
```

Comment:
If we're trying to encourage the use of Bundler everywhere, maybe we should change this to something like

```
bundle exec jekyll --version
bundle show jekyll
```

Maybe that requires a larger refactoring of this document than is in the scope of this PR